vagrant/provisioning/roles/confluent-platform-install/tasks/main.yml

```diff
@@ -81,6 +82,24 @@
     force: yes
     backup: yes
 
+- name: read control-center-production.properties
+  shell: cat /etc/confluent-control-center/control-center-production.properties
+  register: control_center_production_properties_contents
+  changed_when: false
+
+- name: Enable Control Center to work with 1 broker
+  lineinfile:
+    path: /etc/confluent-control-center/control-center-production.properties
+    state: present
+    insertafter: EOF
+    line: "{{ item }}"
+  loop:
+    - "confluent.controlcenter.internal.topics.replication=1"
```

Comment:
Do we want controlcenter in all environments, or only in local development environments?  Most upper environments will have more than one broker.

Reply:
We would use control center in all environments. But the current installation cannot add multiple brokers anyway. This will need to be configurable I guess.